### DISCUSSION TITLE
pre-dev-2021-04 Pre Old Backend Removal Release

### DISCUSSION THREAD

**Discussion by gingerBill** [odin-lang/Odin#917] [General]
Transitionary release for two things:

* Replacing the Old Backend with the New Backend Transition Period #891
* New Odin Versioning System for Pre Version 1.0 #887

## Notes

Please use this release if you are still relying on the old backend for your projects.

Release notes will be published in the release in which the old backend is removed.

<hr /><em>This discussion was created from the release <a href='https://github.com/odin-lang/Odin/releases/tag/pre-dev-2021-04'>dev-2021-04 Pre Old Backend Removal Release</a>.</em>

**Comment by ThisDevDane** [comment]
It's FINALLY happening! :o
